packages/panzer/adapters-stk/src/Panzer_STK_LocalMeshUtilities.cpp

```diff
@@ -254,7 +257,7 @@ buildNodeToCellMatrix(const Teuchos::RCP<const Teuchos::Comm<int> > & comm,
   // This is essentially another way to store cells_to_nodes
   RCP<crs_type> cell_to_node;
   {
-
+    PANZER_FUNC_TIME_MONITOR("Build matrix");
     // The matrix is indexed by (global cell, global node) = local node
     cell_to_node = rcp(new crs_type(cell_map,0));
     cell_to_node->resumeFill();
```

Comment:
The `CrsGraph` and `CrsMatrix` constructors both do the equivalent of calling `resumeFill`, so you don't need to call it here (and shouldn't really).